src/inc/winwrap.h

```diff
@@ -48,6 +47,7 @@
 #include <specstrings.h>
 
 #include "registrywrapper.h"
+#include "winwrappers.h"
```

Comment:
Given that the current file is called winwrap.h, it is the right place to have/define various kind of wrappers. An example is registrywrapper.h right before your inclusion above.

The wrappers you are adding are longfile name specific, it will be good to have the name reflect the intent.